addons/account/migrations/12.0.1.1/post-migration.py

```diff
@@ -167,14 +168,17 @@ def fill_account_move_reverse_entry_id(env):
             'domain': [('auto_reverse', '=', True),
                        ('reverse_date', '=', '2999-12-31')],
         })
+        moves_linked_to_reversals += all_moves.filtered(
+            lambda m: m.reverse_entry_id)
     else:
         installed_langs = env['res.lang'].search([])  # search only active
         for lang in installed_langs:
             reversal_text = env['ir.translation']._get_source(
                 name=False, types='code', lang=lang.code,
                 source='reversal of: ')
             reversal_moves = all_moves.filtered(
-                lambda m: m.ref and m.ref.startswith(reversal_text))
+                lambda m: m.ref and m.ref.startswith(reversal_text)
+                and m not in moves_linked_to_reversals)
```

Comment:
here `moves_linked_to_reversals` will be always empty, so it's irrelevant add `m not in moves_linked_to_reversals`